docs/docs/overview/quickstart/get-started-with-React.md

```diff
@@ -0,0 +1,230 @@
+
+![Using custom variables in Novu](https://res.cloudinary.com/dxc6bnman/image/upload/v1686742931/guides/Screenshot_2023-05-28_at_3.36.10_AM_o6oosa.png)
+
+Feel free to add only text for now and rename the notification template to `quickstart`. It automatically creates a slug-like Identifier that will be needed in later steps to trigger a notification.
+
+![Creating a notification template from Novu dashboard](https://res.cloudinary.com/dxc6bnman/image/upload/v1686743094/guides/Screenshot_2023-05-28_at_3.40.01_AM_ulmqvm.png)
+
+Now, we’ll learn how to create subscribers on Novu - Recipients of Notifications!
+
+## Create a subscriber
+
+If you click “Subscriber” on the left sidebar of [Novu dashboard](https://web.novu.co/subscribers), you’ll see the subscriber list. By default, there will only be one subscriber as you’re automatically added as a subscriber when you sign up for Novu:
+
+![Subscribers page on the Novu dashboard](https://res.cloudinary.com/dxc6bnman/image/upload/v1685465978/guides/subscribers_1_opmw0s.png)
+
+Now, let’s create a subscriber on Novu. After creating a subscriber, we’ll trigger notification to this subscriber. Subscribers are identified by a unique `subscriberId`.
+
+With Novu, you can create a subscriber using any of its SDKs (Node.js, PHP, .NET, Go, Ruby, Python and Kotlin). The code to create a subscriber in Novu is:
+
+```jsx
+import { Novu } from '@novu/node';
+
+const novu = new Novu('<YOUR_NOVU_API_KEY>');
+
+await novu.subscribers.identify('123', {
+  firstName: 'Sumit',
+  lastName: 'Saurabh',
+  returnUser: true,
+});
+```
+
+You can get your API key from the Novu dashboard. Replace `YOUR_NOVU_API_KEY_HERE` with it. Now, if you’ll go to the Novu dashboard, you shall see the subscriber we created above with `subscriberId` of `123`.
+
+You can also update information about an already existing subscriber using the `subscriber.update` method as shown below:
+
+```jsx
+import { Novu } from '@novu/node';
+
+const novu = new Novu('<YOUR_NOVU_API_KEY>');
+
+await novu.subscribers.update('123', {
+  firstName: 'Saurabh', // new first name
+  lastName: 'Sumit', // new last name
+});
+```
+
+## Using Novu in a React app
+
+We have already installed the Novu notification center package above. To use it in an app, simply import it and add the component in your React app as follows:
+
+```jsx
+import React, { useState } from 'react';
+import {
+  NovuProvider,
+  PopoverNotificationCenter,
+  NotificationBell,
+} from '@novu/notification-center';
+import axios from 'axios';
+
+function App() {
+  const [description, setDescription] = useState('');
+  function onNotificationClick(message) {
+    // your logic to handle the notification click
+    if (message?.cta?.data?.url) {
+      window.location.href = message.cta.data.url;
+    }
+  }
+  const onClickHandler = async (e) => {
+    e.preventDefault();
+    const response = await axios.post('http://localhost:3000/sendNotif/create', { description });
```

Comment:
I do not understand this endpoint. Is it a custom made by you in an own API?

Reply:
Yes. I made this end point in the back-end I wrote. Shall I make any changes here?

Reply:
@sumitsaurabh927 please remove the entire onclickHandler function.

Reply:
done in the latest commit